src/utils_social.js

```diff
@@ -500,6 +508,22 @@ try {
  *   youtubes: String[],
  * }
  * ```
+ * @typedef SocialHandles
+ * @property {String[]} emails
+ * @property {String[]} phones
+ * @property {String[]} phonesUncertain
+ * @property {String[]} linkedIns
+ * @property {String[]} twitters
+ * @property {String[]} instagrams
+ * @property {String[]} facebooks
+ * @property {String[]} youtubes
+ */
+
+// TODO yin: Typescript MAY NOT take the `data` param of parseSocialHandles kindly. Find a proper solution for that.
```

Comment:
What does that mean?

Reply:
Removed. I confirmed that this works with ts.